vllm_ascend/ops/triton/fla/fused_qkvzba_split_reshape.py

```diff
@@ -10,10 +10,7 @@
 # ruff: noqa: E501
 # mypy: ignore-errors
 import torch
-from vllm.triton_utils import HAS_TRITON, tl, triton
-
-if HAS_TRITON:
-    import torch_npu._inductor  # noqa: F401
+from vllm.triton_utils import tl, triton
```

Comment:
It seems we need to add a check `Forbid import torch_npu._inductor in vllm_ascend/ops/triton/`
https://github.com/vllm-project/vllm/blob/main/.pre-commit-config.yaml#L132